file-formats/cfdf/examples/yy1_aff_example.cfdf.json

```
@@ -0,0 +1,202 @@
+{
+  "formatVersion": "1 ",
```

@ch-holzer was the space after number 1 introduced manually? Otherwise you might have a look into your object handler.
```suggestion
  "formatVersion": "1",
```

---

Thanks for points this out. The issue was related to the type of the internal constant.